testsuite/test_gpg_decrypt.py

```diff
@@ -0,0 +1,105 @@
+import tempfile
+import json
+from pathlib import Path
+
+from email.message import EmailMessage
+from email.headerregistry import Address
+
+import gnupg
+import pytest
+
+from snoop import pgp
+from django.conf import settings
+
+# http://www.saltycrane.com/blog/2011/10/python-gnupg-gpg-example/
+# If it stalls:     ` $ sudo apt-get install rng-tools `
+
+# To manually remove the passphrase from a key:
+# http://www.cyberciti.biz/faq/linux-unix-gpg-change-passphrase-command/
+
+@pytest.yield_fixture
+def patch_gpg_to_temp_dir(monkeypatch):
+    with tempfile.TemporaryDirectory() as tmp:
+        gpg = gnupg.GPG(gnupghome=tmp, gpgbinary='gpg')
+        monkeypatch.setattr(pgp, 'GPG', gpg)
+        monkeypatch.setattr(settings, 'SNOOP_GPG_HOME', '/tmp')
+        monkeypatch.setattr(settings, 'SNOOP_GPG_BINARY', 'gpg')
+        yield
+
+def test_decryption(patch_gpg_to_temp_dir):
+
+    def create_profile(gpg, name, email, passphrase):
```

Comment:
Let's refactor this into a single pytest fixture called `gpg_home` that yields a `Path` to the folder.